app/controllers/application_controller.rb

```diff
@@ -35,6 +35,10 @@ def cookies_preference_set?
     cookies["consented-to-cookies"].present?
   end
 
+  def cookie_preferences_show?
+    controller_name != "cookies_preferences" && !cookies_preference_set?
```

Comment:
I think it might be nicer to have this just test for `!cookies_preference_set?`, and then override this method to always return false in `CookiesPreferencesController` (rather than checking for the name of a controller).

Reply:
if thats the case ill remove this method. tbh didnt question really why `controller_name != "cookies_preferences"` was there, just a copy and paste from last abtest!

Reply:
If it was in the application layout, it'll be because that's the only way the layout will know which controller it's rendered from. Here we're in the controller to begin with, so can do it more cleanly 🧹